cognite_toolkit/_cdf_tk/loaders/_resource_loaders/transformation_loaders.py

```diff
@@ -482,6 +482,19 @@ def dump_id(cls, id: str) -> dict[str, Any]:
             "destination": destination,
         }
 
+    def _are_equal(
+        self,
+        local: TransformationNotificationWrite,
+        cdf_resource: TransformationNotification,
+        return_dumped: bool = False,
+    ) -> bool | tuple[bool, dict[str, Any], dict[str, Any]]:
+        local_dumped = local.dump()
+        cdf_dumped = cdf_resource.as_write().dump()
+        cdf_dumped.pop("transformationId")
+        cdf_dumped["transformationExternalId"] = cdf_resource.transformation_external_id
+
+        return self._return_are_equal(local_dumped, cdf_dumped, return_dumped)
+
```

Comment:
This feels a bit off, but I don't see any other ways to compare the cdf resource and the yaml resource?